sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusSessionProcessor.cs

```diff
@@ -145,6 +145,7 @@ protected ServiceBusSessionProcessor()
         protected internal virtual async Task OnProcessSessionMessageAsync(ProcessSessionMessageEventArgs args)
         {
             await InnerProcessor.OnProcessSessionMessageAsync(args).ConfigureAwait(false);
+            args.EndExecutionScope();
```

Comment:
Should we consider putting this in a finally block so that we don't potentially miss it in the case of an exception?